src/Confluent.Kafka/ConsumerBuilder.cs

```diff
@@ -159,7 +159,7 @@ public ConsumerBuilder(IEnumerable<KeyValuePair<string, string>> config)
         {
             if (this.StatisticsHandler != null)
             {
-                throw new InvalidOperationException("Statistics handler may not be specified more than once.");
+                this.StatisticsHandler += statisticsHandler;
             }
             this.StatisticsHandler = statisticsHandler;
```

Comment:
You should just be able to say `this.StatisticsHandler += statisticsHandler;` in all cases.

Especially since, as the code currently is, when `this.StatisticsHandler` is not null, we append to it and then overwrite the handler with the incoming handler.

Reply:
good catch thanks